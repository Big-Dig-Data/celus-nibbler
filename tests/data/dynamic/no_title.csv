Platform,YOP,Metrics,2020-01,2020-02,2020-03
"P1","2015","Sessions",1,2,3
"P1","2015","Denied",0,1,0
"P1","2013","Sessions",4,5,6
"P1","2013","Denied",3,0,2
