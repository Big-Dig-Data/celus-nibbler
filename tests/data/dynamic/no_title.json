{
  "kind": "non_counter.date_based",
  "version": 1,
  "parser_name": "no_title",
  "data_format": {
    "name":"simple_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [],
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "Platform",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      },
      {
        "pattern": "YOP",
        "coord": {
          "row": 0,
          "col": 1
        },
        "kind": "regex"
      },
      {
        "pattern": "Metrics",
        "coord": {
          "row": 0,
          "col": 2
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 0,
                      "col": 3
                    },
                    "direction": "right"
                },
                "role": "date"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 3
          },
          "direction": "right"
        }
      },
      "titles": {
        "source": {
          "value": null
        }
      },
      "metrics": {
        "source": {
          "coord": {
            "row": 1,
            "col": 2
          },
          "direction": "down"
        }
      },
      "title_ids": [],
      "dimensions": [
        {
          "name": "YOP",
          "source": {
            "coord": {
              "row": 1,
              "col": 1
            },
            "direction": "down"
          },
          "required": true
        },
        {
          "name": "Platform",
          "source": {
            "coord": {
              "row": 1,
              "col": 0
            },
            "direction": "down"
          },
          "required": true
        }
      ],
      "kind": "non_counter.date_based"
    }
  ]
}
