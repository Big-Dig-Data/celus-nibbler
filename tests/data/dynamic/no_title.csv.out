"2020-01-01","2020-01-31","","","","","","","","Platform:P1|YOP:2015","Sessions","1"
"2020-02-01","2020-02-29","","","","","","","","Platform:P1|YOP:2015","Sessions","2"
"2020-03-01","2020-03-31","","","","","","","","Platform:P1|YOP:2015","Sessions","3"
"2020-01-01","2020-01-31","","","","","","","","Platform:P1|YOP:2015","Denied","0"
"2020-02-01","2020-02-29","","","","","","","","Platform:P1|YOP:2015","Denied","1"
"2020-03-01","2020-03-31","","","","","","","","Platform:P1|YOP:2015","Denied","0"
"2020-01-01","2020-01-31","","","","","","","","Platform:P1|YOP:2013","Sessions","4"
"2020-02-01","2020-02-29","","","","","","","","Platform:P1|YOP:2013","Sessions","5"
"2020-03-01","2020-03-31","","","","","","","","Platform:P1|YOP:2013","Sessions","6"
"2020-01-01","2020-01-31","","","","","","","","Platform:P1|YOP:2013","Denied","3"
"2020-02-01","2020-02-29","","","","","","","","Platform:P1|YOP:2013","Denied","0"
"2020-03-01","2020-03-31","","","","","","","","Platform:P1|YOP:2013","Denied","2"
